tldw_Server_API/app/core/Agent_Client_Protocol/adapters/mcp_transports/sse.py

```
@@ -0,0 +1,292 @@
+
+        SSE events are separated by blank lines. Each event consists of
+        ``event:`` and ``data:`` fields.  Returns ``(event_type, data)``
+        when a blank line (event boundary) is encountered and there is a
+        complete event buffered.  Returns ``None`` otherwise.
+        """
+        line = line.rstrip("\r\n")
+
+        if line.startswith("event:"):
+            self._sse_event_type = line[len("event:"):].strip()
+            return None
+        elif line.startswith("data:"):
```

Ruff (0.15.6)

[warning] Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)